network_management/vpc_flow_logs_config/interconnect_attachment_basic/main.tf

```diff
@@ -0,0 +1,26 @@
+/**
+* Copyright 2024 Google LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+# [START vpcflowlogs_create_basic_vpc_flow_logs_config_over_interconnect_attachment]
+# google_network_management_vpc_flow_logs_config.vpc_fl_config will be created
+resource "google_network_management_vpc_flow_logs_config" "vpc_fl_config" {
+  interconnect_attachment = "projects/example_project/regions/us-central1/interconnectAttachments/example_interconnect"
+  location                = "global"
+  project                 = "example_project"
+  provider                = google-beta
```

Comment:
For clarity on samples, this field should be placed at the top of the arguments so it's clear it's still in beta.

Reply:
The linter sorted the arguments alphabetically automatically.
Moved it to the top again and I'll see if the lint-checker doesn't mind it.

Reply:
I am unable to test it myself so please run the linter and the CI tests again on the Pull Request